src/components/grid-aware/ThreeParagraphBlock/ThreeParagraphBlock.js

```diff
@@ -58,20 +95,41 @@ const ThreeParagraphBlock = ({
   paragraph1,
   paragraph2,
   paragraph3,
-  image1,
-  image2,
-  image3,
+  leftTopImage,
+  leftBottomImage,
+  rightImage,
   ctaTitle,
   ctaButtons,
 }) => {
+  let optionalLeftTopImage;
+  let leftBottomImageWrapperClassName = s.leftBottomImageWrapper;
+
+  if (leftTopImage) {
+    optionalLeftTopImage = (
+      <div className={s.leftTopImageWrapper}>
+        <img
+          className={s.image}
+          src={leftTopImage.url}
+          alt={leftTopImage.alt}
+        />
+      </div>
+    );
+
+    leftBottomImageWrapperClassName += ` ${s.v1}`;
```

Comment:
Could we give the CSS class a more descriptive name? It'll be difficult to remember which is `v1` and `v2`. How about `solo` and `paired`? Feel free to come up with your own names, too.

Reply:
I like `solo` and `paired`. Thanks!